### DISCUSSION TITLE
Issue: Traced runs are not possible to delete after selected?

### DISCUSSION THREAD

**Discussion by iokinpardo** [langchain-ai/langsmith-sdk#424] [Q&A]
### Issue you'd like to raise.

![image](https://github.com/langchain-ai/langsmith-sdk/assets/6673107/a66e9736-ad8a-40ee-91b2-6dce3b3fc271)

I have seen on a previous issue as resolved this for **Test-runs**. See below image.

![image](https://github.com/langchain-ai/langsmith-sdk/assets/6673107/70544265-4bbe-44da-8cb7-3c6f6928ce71)

### Suggestion:

Is this not possible for **Traces** too? This could help having a more clean list of traces if some tests could be removed manually from the list.

Many thanks in advance

**Comment by hinthornw** [comment]
It is not possible for traces, only for tests/projects

**Reply by iokinpardo** [reply]
Many thanks for your fast response. 😀Is this something technically not possible or not desirable due to other reasons-aspects.?

**Reply by hinthornw** [reply]
It's technically possible but not really desirable for production logs to be deleted on a row level in most cases
